@navikt/core/react/src/alert/Alert.tsx

```diff
@@ -67,6 +67,7 @@ const Alert = forwardRef<HTMLDivElement, AlertProps>(
   ) => (
     <div
       {...rest}
+      role="alert"
```

Comment:
En hakket mer fleksibel løsning kan være å ta en `role`-prop som vi defaulter til `"alert"`. Venter uansett på svar fra Sarah.

Reply:
Godt poeng. Det gjør MUI også.

Reply:
Hmm...det er egentlig ikke hvordan `role="alert"` fungerer.  Elementer med `role="alert"` fungerer som containere for feilmeldinger som kan dukke opp dynamisk.  For å fungere, må elementet med `role="alert"` allerede være på plass i DOMen fra siden lastes inn, så kan det injiseres innhold i elementet. Det injiserte innholdet blir da lest opp umiddelbart og avbryter alt annet som blir lest opp.  Se f.eks. https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html for mer info.  

Brukes komponenten dynamisk?